server/logs/lib.lua

```diff
@@ -9,33 +9,61 @@ function Logs.findContext()
 	return info
 end
 
-local function isValidLogLevel(level)
-	if level == "info" or "warn" or "error" then
-		return true
-	end
-
-	return false
-end
-
 local function sendHttpRequest(data)
 	PerformHttpRequest(apiUrl, function() end, "POST", json.encode(data), {
 		["Authorization"] = apiKey,
 		["Content-Type"] = "application/json",
 	})
 end
 
-function Logs:LogMessage(level, message, metadata)
-	local isValid = isValidLogLevel(level)
-	if not isValid then
-		print("Log level is not valid. Use 'info', 'warn' or 'error'.")
+local function getIdentifiers(playerSource)
+	local identifiers = GetPlayerIdentifiers(playerSource)
+	local data = {}
+	for i = 1, #identifiers do
+		--split the identifier at the colon
+		local identifier = identifiers[i]
+		local _, _, idType, id = string.find(identifier, "([^:]+):(.+)")
+		data[idType] = id
+	end
+
+	return data
+end
+
+---@alias logType string | '"info"' | '"warn"' | '"error"' | 'any' | string
+
+---@class LogOptions
```

Comment:
Looking at this now, this might be better approach than what I previously proposed. Let's keep everything in the metadata table, including the player data.